tasks/elasticsearch-security.yml

```diff
@@ -17,27 +17,27 @@
   tags:
     - certificates
```

Comment:
The **certificate** tag should be mintioned while importing the task file to get it worked e.g.
```yaml
- import_tasks: elasticsearch-security.yml
  when:
    - elasticsearch_security | bool
    - elastic_variant == "elastic"
  tags:
    - security
    - certificates
```

Reply:
Good catch! I had some problems with running tasks in the past. Thanks.

Reply:
What is the purpose of the tags here? For debugging use for yourself or to let the users be able to run a part of the role? When the second one is the right one, I would write something about this functionality in the README :)

Reply:
It is only a suggestion :) when you do not see it right, I can close the conversation

Reply:
I did not see this issue. So we can ignore the update of the documentation now.

The **security** tag is used while the import of **elasticsearch-security.yml** but I do not see any usage of this tag in the tasks of this file.